src/main/java/by/home/port/LoadProcess.java

```diff
@@ -0,0 +1,14 @@
+package by.home.port;
+
+public class LoadProcess implements Process{
+    @Override
+    public void activate(Ship ship, Dock dock) {
+        System.out.println(ship.toString() + " has been loading in the dock: " + dock.getId());
+        try {
+            Thread.sleep(1000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```

Comment:
logging? you may create a helper class where you put this hepler method with sleep in order not to duplicate this try eveywhere